README.md

```diff
@@ -40,19 +43,19 @@ In addition to modifying the json files mentioned above, the get_data function i
 
 The sample is configured using the file [appsettings.placeholder.json](appsettings.placeholder.json). Before editing, rename this file to `appsettings.json`. This repository's `.gitignore` rules should prevent the file from ever being checked in to any fork or branch, to ensure credentials are not compromised.
 
-The application can be configured to send to any number of endpoints specified in the endpoints array within appsettings.json. In addition, there are three types of endpoints: [OCS](#ocs-endpoint-configuration), [EDS](#eds-endpoint-configuration), and [PI](#pi-endpoint-configuration). Each of the 3 types of enpoints are configured differently and their configurations are explained in the sections below.
+The application can be configured to send to any number of endpoints specified in the endpoints array within appsettings.json. In addition, there are three types of endpoints: [ADH](#adh-endpoint-configuration), [EDS](#eds-endpoint-configuration), and [PI](#pi-endpoint-configuration). Each of the 3 types of enpoints are configured differently and their configurations are explained in the sections below.
 
-### OCS Endpoint Configuration
+### ADH Endpoint Configuration
 
-An OMF ingress client must be configured. On our [OSIsoft Learning](https://www.youtube.com/channel/UC333r4jIeHaY-rGgMjON54g) Channel on YouTube we have a video on [Ceating an OMF Connection](https://www.youtube.com/watch?v=52lAnkGC1IM).
+An OMF ingress client must be configured. On our [AVEVA Learning](https://www.youtube.com/channel/UC333r4jIeHaY-rGgMjON54g) Channel on YouTube we have a video on [Creating an OMF Connection](https://www.youtube.com/watch?v=52lAnkGC1IM).
```

Comment:
do we know that "AVEVA Learning Channel" will be the new name?

Reply:
No that is just an assumption, I could leave it at OSI until I know more

Reply:
Aveva learning channel sounds like a far greater scope than just the OSIsoft business unit, so that's mainly my thought there. I dont think it really matters too much, but personally I'd lean towards keeping it for now until they decide their new name

Reply:
Okay, that's a good point, I'll keep OSIsoft as this is mostly OCS -> ADH more so that OSIsoft -> AVEVA